test/orange/orangeinp/CsgObject.test.cc

```diff
@@ -311,6 +311,63 @@ TEST_F(CsgObjectTest, subtraction)
     EXPECT_VEC_EQ(expected_trans_strings, transform_strings(u));
 }
 
+TEST_F(CsgObjectTest, subtraction_atlas)
```

Comment:
Maybe you can add a high-level comment on what this test does and why it is necessary.